### ISSUE TITLE
ClassCastException when calling getPatternsFor

### ISSUE THREAD

**Issue by kawashirov** [SirEdvin/UnlimitedPeripheralWorks#42]
Hello there!
I am playing **StaTech Industry** modpack right now and got a problem:
If there is a pattern with any fluid on AE2 network,
if I call `getPatternsFor` for that pattern,
It will not return expected result,
but will crash (the program on CC, not the game) with following error (copied from the server log):
```
[20:52:24] [Server thread/ERROR]: Error running task
java.lang.ClassCastException: class java.lang.Double cannot be cast to class java.lang.Integer (java.lang.Double and java.lang.Integer are in module java.base of loader 'bootstrap')
	at site.siredvin.peripheralworks.integrations.ae2.MENetworkBlockPlugin.getPatternsFor(MENetworkBlockPlugin.kt:393) ~[peripheralworks-fabric-1.19.2-0.2.12.jar:?]
	at site.siredvin.peripheralworks.integrations.ae2.MENetworkBlockPlugin$cc$getPatternsFor18.apply(CC generated method) ~[?:?]
	at dan200.computercraft.core.asm.PeripheralMethod.lambda$static$0(PeripheralMethod.java:21) ~[cc-restitched-1.101.2.jar:?]
	at dan200.computercraft.core.lua.LuaContext.lambda$issueMainThreadTask$0(LuaContext.java:34) ~[cc-restitched-1.101.2.jar:?]
	at dan200.computercraft.core.computer.MainThreadExecutor.execute(MainThreadExecutor.java:143) ~[cc-restitched-1.101.2.jar:?]
	at dan200.computercraft.core.computer.MainThread.executePendingTasks(MainThread.java:157) ~[cc-restitched-1.101.2.jar:?]
	at dan200.computercraft.shared.proxy.ComputerCraftProxyCommon.lambda$registerHandlers$3(ComputerCraftProxyCommon.java:105) ~[cc-restitched-1.101.2.jar:?]
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerTickEvents.lambda$static$0(ServerTickEvents.java:40) ~[fabric-lifecycle-events-v1-2.2.4+1b46dc7890-ba8a0595c21e6ba2.jar:?]
	at net.minecraft.server.MinecraftServer.handler$cac000$fabric-lifecycle-events-v1$onStartTick(MinecraftServer.java:8316) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:819) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:665) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:1583) [?:?]
```

**StaTech Industry** modpack version: 1.1.13
Minecraft version: 1.19.2
Fabric version: 0.15.7
AE2 version: 12.9.9 (appliedenergistics2-fabric-12.9.9.jar)
Unlimited Peripheral Works version: 0.2.12 (peripheralworks-fabric-1.19.2-0.2.12.jar)
Peripheralium version: 0.4.23 (peripheralium-fabric-1.19.2-0.4.23.jar)
CC:Restitched version: 1.101.2 (cc-restitched-1.101.2.jar)

Simple Example: making dye from benzene in Modern Industrialization:
![javaw_PvAgZSNoeM](https://github.com/user-attachments/assets/3e7d253a-3cfb-4905-95dd-2a1f81c7f67a)
![javaw_GktVs5VqHy](https://github.com/user-attachments/assets/5e40ffaf-26ff-4986-924a-3cc5c59ac495)

**Comment by kawashirov** [comment]
Looks like shit happens with those `as Int` casts but I not sure - I don't rly know Kotlin.
![msedge_oYonopDIir](https://github.com/user-attachments/assets/1174a205-edfa-439c-9b81-5e7fc7f32d28)

**Comment by SirEdvin** [comment]
Fixed in 1.4.4 release
